adminsortable/templates/adminsortable/change_list.html

```diff
@@ -29,6 +29,23 @@
 <script src="{% static 'adminsortable/js/jquery.ui.touch-punch.min.js' %}"></script>
 {% include 'adminsortable/csrf/jquery.django-csrf.html' with csrf_cookie_name=csrf_cookie_name %}
 <script src="{% static 'adminsortable/js/admin.sortable.js' %}"></script>
+
+<script type="text/javascript">
+(function($) {
+    $(document).ready(function($) {
+        var url = window.location.href;
+        var urlParts = url.split('?');
+        var cnangeListUrl = $('a#return-to-changelist').attr('href')

```

Comment:
You have a small typo here. I think you meant `changeListUrl` instead of `cnangeListUrl`

Reply:
yep. thank you

Reply:
@alsoicode what do you think about this pr ? is it useful ?